production/scrapers/pennsylvania_bi_deaths.R

```diff
@@ -0,0 +1,111 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+pennsylvania_bi_deaths_pull <- function(x, wait = 10){
+    # scrape from the power bi iframe directly
+    y <- "https://app.powerbigov.us/view?r=" %>%
+        str_c(
+            "eyJrIjoiMTcyY2I2MjMtZjJjNC00NjNjLWJjNWYtNTZlZWE1YmRkYWYwIiwidCI",
+            "6IjQxOGUyODQxLTAxMjgtNGRkNS05YjZjLTQ3ZmM1YTlhMWJkZSJ9",
+            "&pageName=ReportSectionc6adf1f0d7011b2ed62c")
+    
+    remDr <- RSelenium::remoteDriver(
+        remoteServerAddr = "localhost",
+        port = 4445,
+        browserName = "firefox"
+    )
+    
+    del_ <- capture.output(remDr$open())
+    remDr$navigate(y)
+    
+    Sys.sleep(wait)
+    
+    raw_html <- xml2::read_html(remDr$getPageSource()[[1]])
+    
+    is_covid_deaths <- raw_html %>%
+        rvest::html_nodes(xpath="//div[@class='preTextWithEllipsis']") %>%
+        rvest::html_text() %>%
+        str_detect("(?=.*Inmate)(?=.*Death)(?=.*COVID)") %>%
+        any()
+    
+    if(!is_covid_deaths){
+        warning("Page structure may have changed please inspect.")
+    }
+    
+    raw_html
+}
+
+pennsylvania_bi_deaths_restruct  <- function(x){
+    val_sr_str <- "//text[@class='label' and contains(@transform,'translate')]"
+    lab_sr_str <- "//text[@class='setFocusRing']//title"
+
+    tibble(
+        Name = raw_html %>%
```

Comment:
This should be `x` I think! (same thing right below)

Reply:
Thanks for catching!